src/pages/lock-exception/LockException.js

```diff
@@ -287,7 +291,7 @@ class LockException extends Page {
         if (userAction || (!this.props.loading && !this.props.loaded)) {
             // Keep the previous message visible (p.e. deleting lock exception || add lock exception)
             this.context.updateAppState(
-                this.props.deleteInProgress || this.props.addInProgress
+                this.deleteInProgress || this.addInProgress
```

Comment:
I've implemented `deleteInProgress` and `addInProgress` as instance properties because they were being mutated directly. I don't think we should directly mutate props.